model/host/hostutil.go

```diff
@@ -230,16 +230,22 @@ func (h *Host) jasperExtractedFileName(config evergreen.JasperConfig) string {
 // BootstrapScript creates the user data script to bootstrap the host.
 func (h *Host) BootstrapScript(config evergreen.JasperConfig) string {
 	if h.Distro.IsWindows() {
-		cmds := h.FetchJasperCommandWithPath(config, "/bin")
+		cmds := []string{
+			h.FetchJasperCommandWithPath(config, "/bin"),
+			h.ForceReinstallJasperCommand(config),
+		}
 		// PowerShell nested quotation marks are handled by using two quotation
 		// marks.
-		quotedCmds := strings.Replace(cmds, "'", "''", -1)
+		quotedCmds := make([]string, 0, len(cmds))
+		for _, cmd := range cmds {
+			quotedCmds = append(quotedCmds, strings.Replace(cmd, "'", "''", -1))
+		}
```

Comment:
does it make sense to put this in some kind of util function?

Reply:
It seems like this is very specific to Powershell only, so unless there's some future use case for writing Powershell scripts programmatically, I think this is just a one-off thing.